openreview/conference/templates/recruitReviewersProcess.js

```diff
@@ -12,56 +13,61 @@ function() {
   var user = decodeURIComponent(note.content.user);
   var hashKey = or3client.createHash(user, HASH_SEED);
 
-  if (hashKey === note.content.key) {
-    if (note.content.response === 'Yes') {
-      console.log('Invitation replied Yes');
-      //if a user is in the declined group, remove them from that group and add them to the reviewers group
-      var text = 'Thank you for accepting the invitation to be a ' + REVIEWER_NAME + ' for ' + SHORT_PHRASE + '.\n';
-      text += 'The ' + SHORT_PHRASE + ' program chairs will be contacting you with more information regarding next steps soon. ';
-      text += 'In the meantime, please add noreply@openreview.net to your email contacts to ensure that you receive all communications.\n\n';
-      text += 'If you would like to change your decision, please click the Decline link in the previous invitation email.';
+  var invitedGroupP = or3client.or3request(or3client.grpUrl + '?id=' + REVIEWERS_INVITED_ID, {}, 'GET', token);
```

Comment:
can you add the parameter `member` to this call and check the result is empty or not?

Reply:
As discussed, we are not making this suggested change right now because of the issue https://github.com/openreview/openreview/issues/2156. I can make the change once this gets resolved.